test/e2e/registration_service_test.go

```diff
@@ -306,7 +306,7 @@ func (s *registrationServiceTestSuite) TestSignupOK() {
 		s.assertGetSignupStatusPendingApproval(identity.Username, token)
 
 		// Attempt to create same usersignup by calling post signup with same token should return an error
-		mp := invokeEndpoint(s.T(), "POST", s.route+"/api/v1/signup", token, "", http.StatusInternalServerError)
+		mp := invokeEndpoint(s.T(), "POST", s.route+"/api/v1/signup", token, "", http.StatusBadRequest)
```

Comment:
Should be 409, right?

Reply:
Yeah, didn't get to fixing it as fast because i noticed another error, for some reason the `re-registering new user with pre-existing ban` test now expects a "BadRequest" error instead of "Forbidden"... i'm not too sure what caused that change when looking at my code change, but i'll investigate that. Here is the change for the conflict: 6d59900c16a2d2577d05e07ce7577dd0d04e0054